website/versioned_docs/version-1.0.0/contracts/deploying-a-contract.md

```diff
@@ -13,6 +13,7 @@ After successfully [installing Substrate](getting-started/installing-substrate.m
 ```bash
 substrate --dev
 ```
+> **Note:** For compatibility be sure you are using [stable, known working version of the Substrate Node Template for this tutorial, `contract` branch](https://github.com/shawntabrizi/substrate-package/tree/contract) 
```

Comment:
Thanks. 
This page is based on substrate full installation which is for substrate master branch. We need to improve this page later. 
Your change is not very helpful for newcomers since there is no instruction around ink! in `substrate-package`.

Reply:
That would require testing of each chapter if they are ok. 
If there are expectations of soon™️ upcoming stable v2.0, I would rather wait and keep telling that they have to change branch to `contract`.